homeassistant/components/zoneminder/sensor.py

```diff
@@ -95,53 +97,39 @@ def setup_platform(
 class ZMSensorMonitors(SensorEntity):
     """Get the status of each ZoneMinder monitor."""
 
-    def __init__(self, monitor):
+    def __init__(self, monitor: Monitor) -> None:
         """Initialize monitor sensor."""
         self._monitor = monitor
-        self._state = None
-        self._is_available = None
-
-    @property
-    def name(self):
-        """Return the name of the sensor."""
-        return f"{self._monitor.name} Status"
-
-    @property
-    def native_value(self):
-        """Return the state of the sensor."""
-        return self._state
-
-    @property
-    def available(self):
-        """Return True if Monitor is available."""
-        return self._is_available
+        self._attr_available = False
```

Comment:
Same here, `available` should not be returning `None`, so I have adjusted to equivalent falsy value, but maybe `add_entities(sensors, True)` would be better.